Pipfile

```diff
@@ -4,8 +4,8 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-websocket-client= "0.57.0"
-urllib3 = "1.26.5"
+websocket-client= "1.1.0"
+urllib3 = "1.26.6"
 httpx = {extras = ["http2"], version = "0.19.0"}
 
```

Comment:
There's also requirements.txt  - which is generated by pipfile, so I think that changes should be applied there as well -accordingly to docs it's generated through pipenv lock --requirements